Maintainers.txt

```diff
@@ -647,6 +647,7 @@ M: Ray Ni <ray.ni@intel.com> [niruiyu]
 R: Jiaxin Wu <jiaxin.wu@intel.com> [jiaxinwu]
 R: Dun Tan <dun.tan@intel.com> [td36]
 R: Khor Swee Aun <swee.aun.khor@intel.com> [sweeaun]
+R: Eduardo Cuevas Farfan <eduardo.cuevas.farfan@intel.com> [ecuevasf]

```

Comment:
nit: The commit message is currently written: "Add **ecuevas** as reviewer for StandaloneMmPkg and UefiCpuPkg" whereas the GitHub ID actually is **ecuevasf**. Can you please update the commit message so they're consistent?